ykcapi/src/lib.rs

```diff
@@ -124,36 +124,27 @@ struct AOTVar {
     bbidx: usize,
     instridx: usize,
     fname: *const i8,
+    sfidx: usize,
 }
 
-/// Address and length of a vector containing AOTVars. Mirrors the struct defined in
+/// Address and length of a vector. Mirrors the struct defined in
 /// ykllvmwrap/jitmodbuilder.cc.
 #[derive(Debug)]
 #[repr(C)]
-pub struct AOTMap {
+pub struct VecLen {
```

Comment:
This isn't an ideal struct name -- it's not the length of a vector, but an actual vector! Why not call it `AOTVars` or `AOTVarsVec` or similar?

Reply:
Because I am now using it for stackframes too. So basically anytime I need to pass a vector and its length from C into Rust I need to use this struct. I'm open for suggestions. Maybe `VecPtrAndLen`?

Reply:
Why not call it `CVec` or similar?

Reply:
Fixed in b83cf7c2